go/teams/audit.go

```diff
@@ -333,15 +364,20 @@ func (a *Auditor) scheduleProbes(m libkb.MetaContext, probes map[keybase1.Seqno]
 		m.CDebugf("no more probes needed; range was only %d, and we did %d", rng, probesInRange)
 		return nil, nil
 	}
+	currentProbes := make(map[keybase1.Seqno]bool)
```

Comment:
Make a local map to catch dupes, don't mutate the one we were passed in. That was a bad idea. Can't say I wasn't warned!